### DISCUSSION TITLE
Milter headers - add x-spam-status for adding SpamAssassin header style

### DISCUSSION THREAD

**Discussion by srsr** [rspamd/rspamd#4389] [Q&A] [answered]
Hello,

I'm using extended_spam_headers = true; which is working great.

Now I want to add "x-spam-status" function to add spamassasin header style.

Here is my milter_headers.conf that unfortunately desn't work at all... ( even the rspamd spam headers are not present )

```
extended_spam_headers = true;
use = ["X-Spam-Status"];
routines {
  X-Spam-Status{
    remove = 0;
  }
}
```

This is my understanding of the x-spam-status documentation ( https://rspamd.com/doc/modules/milter_headers.html#x-spam-status )

_SpamAssassin-style X-Spam-Status header indicating spam status.
header = "X-Spam-Status";
remove = 0;_

Could someone tell me what I miss?
Thank a lot
Samuel

**Comment by srsr** [comment]
I Also tried :
```
extended_spam_headers = true;
use = ["X-Spam-Status"];
routines {
   X-Spam-Status {
     header = "X-Spam-Status";
     remove = 0;
   }
}
```
This try make x-spamd-result, x-rspamd-server and x-rspamd-queue-id work again.
this said, it is still missing spammassin header.
:/

**Comment by tokred** [comment, marked as answer]
Just a thought, did you try lowercase? I use this `milter_headers.conf` and the header gets added (no further `routines {}`):

```
use = ["x-spam-status", "x-spamd-bar", "x-virus", "authentication-results"];
authenticated_headers = ["authentication-results"];
extended_spam_headers = true;
```

Regards

**Reply by srsr** [reply]
Thanks a lot Tokred, that was that. 🥇
